### PR TITLE
Fix funcArgNamesDifferent with cppcheck

### PR DESCRIPTION
**Pull request by IhorNehrutsa**

### CODE REVIEW

**Comment by IhorNehrutsa** [comment]
That seems to be all.
Thanks.

**Comment by IhorNehrutsa** [comment]
Up

File: src/OneButton.cpp [hunk 1]
```diff
@@ -257,7 +257,7 @@ void OneButton::_fsm(bool activeLevel)
       _newState(OneButton::OCS_UP);
       _startTime = now; // remember starting time
 
-    } else if ((activeLevel) && (waitTime > _press_ms)) {
+    } else if (waitTime > _press_ms) {
```
Comment (IhorNehrutsa) [inline src/OneButton.cpp:260]: activeLevel is true here

**Comment by mathertel** [comment]
Thanks

**Pull request merged**